core/src/main/scala/kafka/server/MetadataCache.scala

```diff
@@ -25,6 +25,7 @@ import org.apache.kafka.common.{Cluster, Node, TopicPartition, Uuid}
 import org.apache.kafka.server.common.{Features, MetadataVersion}
 
 import java.util
+import scala.collection._
```

Comment:
Is this change a improvement?

Reply:
Yes, it makes it possible to return non immutable maps without copying.